### PR TITLE
etcdctlv3: interactive_v3 del command update

### PR DESCRIPTION
**Pull request by sinsharat**
added examples for del flags prev-kv, prefix and from-key

### CODE REVIEW

File: Documentation/dev-guide/interacting_v3.md [hunk 1]
```diff
@@ -181,6 +192,41 @@ $ etcdctl del foo foo9
...
+```
+
+Here is the command to delete keys having prefix as `zoo`:
+
+```bash
+$ etcdctl del --prefix zoo 
+2 # two keys are deleted
+```
+
+## Delete keys which are greater than or equal to the byte value of the specified key
```
Comment (xiang90) [inline Documentation/dev-guide/interacting_v3.md]: probably remove this title and keep the content in the previous delete section?

File: Documentation/dev-guide/interacting_v3.md [hunk 2]
```diff
@@ -181,6 +192,41 @@ $ etcdctl del foo foo9
 2 # two keys are deleted
 ```
 
+Here is the command to delete key `zoo` with the deleted key value pair returned:
+
+```bash
+$ etcdctl del --prev-kv zoo 
+1   # one key is deleted
+zoo # deleted key
+val # deleted value
```
Comment (xiang90) [inline Documentation/dev-guide/interacting_v3.md]: # the value of the deleted key?

File: Documentation/dev-guide/interacting_v3.md [hunk 3]
```diff
@@ -181,6 +192,41 @@ $ etcdctl del foo foo9
...
+2 # two keys are deleted
+```
+
+## Delete keys which are greater than or equal to the byte value of the specified key
+
+Applications may want to delete keys which are greater than or equal to the byte value of the specified key.
+
+Suppose an etcd cluster already has the following keys:
+
+``` bash
```
Comment (gyuho) [inline Documentation/dev-guide/interacting_v3.md]: extra white space

**Comment by gyuho** [comment]
@sinsharat I've noticed you've cleaning up markdown formats in previous PRs

In this PR, can we also change all

<pre>
``` bash
</pre>

to

<pre>
```bash
</pre>

And

<pre>
```
</pre>
to

<pre>
```bash
</pre>


We should change them all at once.

File: Documentation/dev-guide/interacting_v3.md (hunk 1 above)
Comment (sinsharat) [inline Documentation/dev-guide/interacting_v3.md]: ok will do that.

File: Documentation/dev-guide/interacting_v3.md (hunk 2 above)
Comment (sinsharat) [inline Documentation/dev-guide/interacting_v3.md]: ok will change the comment

**Comment by sinsharat** [comment]
@gyuho Sure will change it in this PR itself.
Thanks!

File: Documentation/dev-guide/interacting_v3.md (hunk 3 above)
Comment (sinsharat) [inline Documentation/dev-guide/interacting_v3.md]: @gyuho Actually this is the format being followed for the entire doc, i.e. to give a space before specifying the keys. Do you want me to remove the line for entire doc or maintain it as per the doc?

Comment (sinsharat) [inline Documentation/dev-guide/interacting_v3.md]: @gyuho Sorry i had mis-understood. you were referring to bash. Will do that for sure.

**Comment by sinsharat** [comment]
@xiang90 @gyuho I have done the changes as per your suggestion. Let me know if anything else is required.
Thank you!

**Comment by xiang90** [comment]
lgtm

**Pull request merged**